src/pages/developing/hardware-back-button.md

```diff
@@ -5,15 +5,15 @@ previousUrl: '/docs/developing/tips'
 
 # Hardware Back Button
 
-## Overview
-
 By default, when a user presses the Android hardware back button, the current view will be popped off the navigation stack. If there are no views to pop, nothing will happen. This guide will show you how to customize this behavior.
```

Comment:
Can we add a brief summary on what the back button is before jumping into this

Reply:
I have https://github.com/ionic-team/ionic-docs/pull/1356/files#diff-118692f5d92350834ff00b783ab8591aR10 right after, should I move it to the top instead?

Reply:
I think something more like what the hardware back button is:

```
The hardware back button is found on most Android devices. In native applications it can be used to close modals, navigate to the previous view, exit an app, and more. By default in Ionic, when the back button is pressed, the current view will be popped off the navigation stack and the previous view will be displayed. If no previous views exist in the navigation stack, nothing will happen.  This guide will show to customize the behavior of the hardware back button.

> The hardware back button refers to the physical back button on an Android device and should not be confused with either the browser back button or `ion-back-button`. The information in this guide only applies to Android devices and is not relevant for iOS devices.
```

Feel free to change up my words, just threw it together.